RegistrationApp/src/main/java/app/User.java

```diff
@@ -0,0 +1,146 @@
+package app;
+
+import java.time.LocalDate;
+import java.util.Objects;
+
+public class User {
+    int id;
+    String username;
+    String password;
+    String lastName;
+    String firstName;
+    LocalDate date;
+    String email;
+    String phoneNumber;
+    String address;
+    boolean isAvailable;
+
+    public int getId() {
+        return id;
+    }
+
+    public void setId(int id) {
+        this.id = id;
+    }
+
+
+    public String getUsername() {
+        return username;
+    }
+
+    public void setUsername(String username) {
+        this.username = username;
+    }
+
+    public String getPassword() {
+        return password;
+    }
+
+    public void setPassword(String password) {
+        this.password = password;
+    }
+
+    public String getLastName() {
+        return lastName;
+    }
+
+    public void setLastName(String lastName) {
+        this.lastName = lastName;
+    }
+
+    public String getFirstName() {
+        return firstName;
+    }
+
+    public void setFirstName(String firstName) {
+        this.firstName = firstName;
+    }
+
+    public LocalDate getDate() {
+        return date;
+    }
+
+    public void setDate(LocalDate date) {
+        this.date = date;
+    }
+
+    public String getEmail() {
+        return email;
+    }
+
+    public void setEmail(String email) {
+        this.email = email;
+    }
+
+    public String getPhoneNumber() {
+        return phoneNumber;
+    }
+
+    public void setPhoneNumber(String phoneNumber) {
+        this.phoneNumber = phoneNumber;
+    }
+
+    public String getAddress() {
+        return address;
+    }
+
+    public void setAddress(String address) {
+        this.address = address;
+    }
+
+    public boolean getIsAvailable() {
```

Comment:
rename it to isAvailible()

Reply:
this is GETTER. Why do we need to rename it?

Reply:
for boolean types you don't need  'get' before isVARIABLE()